argopy/data_fetchers/erddap_data.py

```diff
@@ -24,8 +24,15 @@
 from argopy.stores import httpstore
 from argopy.plotters import open_dashboard
 
-from erddapy import ERDDAP
-from erddapy.utilities import parse_dates, quote_string_constraints
+# Load erddapy according to available version (breaking changes in v0.8.0)
+try:
+    from erddapy import ERDDAP
+    from erddapy.utilities import parse_dates, quote_string_constraints
+except:
+    # >= v0.8.0
+    from erddapy.erddapy import ERDDAP
+    from erddapy.erddapy import _quote_string_constraints as quote_string_constraints
+    from erddapy.erddapy import parse_dates
```

Comment:
Sorry. That should've been 1.0. Going from 0.7 to 0.8 with that kind of refactor was not nice!

Reply:
it's ok !